java-bootcamp-bahia-t1/src/MailListManager.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;

```

Comment:
In real life the mailList should use an email service to send all the notifications to a list of people to be notified